iotilecore/iotile/core/hw/auth/env_auth_provider.py

```diff
@@ -1,226 +1,50 @@
-"""An auth provider that can sign and verify with user keys stored in environment variables."""
-
-import hashlib
-import hmac
-import binascii
 import os
+import binascii
+import struct
+import hmac
+import hashlib
 from iotile.core.exceptions import NotFoundError
-from .auth_provider import AuthProvider
+from .rootkey_auth_provider import RootKeyAuthProvider
 
 
-class EnvAuthProvider(AuthProvider):
-    """Authentication provider that can sign and verify using user keys
+class EnvAuthProvider(RootKeyAuthProvider):
+    """ Key provider implementation that search for keys in environment """
```

Comment:
```suggestion
    """Key provider implementation that search for keys in environment """
```

Style consistency note (minor), most of the code has no space between the opening triple quote and the first word

Reply:
Also want to clarify if the original docstring was meant to be deleted instead of updated

Reply:
The original was meant to be updated